skatepark/theme.json

```diff
@@ -1,7 +1,28 @@
 {
 	"$schema": "https://json.schemastore.org/theme-v1.json",
 	"version": 1,
+	"customTemplates": [
+		{
+			"name": "blank",
```

Comment:
Should we also add the header + footer only template?

Reply:
Yeah, I think so.